embulk-standards/src/main/java/org/embulk/standards/CsvFormatterPlugin.java

```diff
@@ -124,7 +128,7 @@ public void timestampColumn(Column column)
                         private void addDelimiter(Column column)
                         {
                             if (column.getIndex() != 0) {
-                                encoder.addText(",");
+                                encoder.addText(task.getDelimiterChar());
```

Comment:
Getter methods of Task classes are very slow. Can you change the code to cache the String to a local variable?